### ISSUE TITLE
Caused by: java.lang.UnsupportedClassVersionError: com/github/rholder/gradle/acumen/api/AcumenTreeModel : Unsupported major.minor version 51.0

### ISSUE THREAD

**Issue by seregasheypak** [rholder/gradle-view#13]
Do I have to switch smth in Idea JDK? I'm using MacOS

```
Caused by: com.intellij.diagnostic.PluginException: com/github/rholder/gradle/acumen/api/AcumenTreeModel : Unsupported major.minor version 51.0 [Plugin: Gradle View]

	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:130)
	at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2484)
	at java.lang.Class.getDeclaredMethod(Class.java:1971)
	at java.io.ObjectStreamClass.getPrivateMethod(ObjectStreamClass.java:1394)
	at java.io.ObjectStreamClass.access$1700(ObjectStreamClass.java:51)
	at java.io.ObjectStreamClass$2.run(ObjectStreamClass.java:446)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.io.ObjectStreamClass.<init>(ObjectStreamClass.java:421)
	at java.io.ObjectStreamClass.lookup(ObjectStreamClass.java:318)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1115)
	at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1535)
	at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1496)
	at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1413)
	at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1159)
	at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:329)
	at org.gradle.tooling.internal.provider.PayloadSerializer.serialize(PayloadSerializer.java:82)
	at org.gradle.tooling.internal.provider.ProviderConnection.run(ProviderConnection.java:98)
	at org.gradle.tooling.internal.provider.DefaultConnection.run(DefaultConnection.java:171)
	at org.gradle.tooling.internal.consumer.connection.CancellableConsumerConnection$CancellableActionRunner.run(CancellableConsumerConnection.java:105)
	at org.gradle.tooling.internal.consumer.connection.AbstractConsumerConnection.run(AbstractConsumerConnection.java:60)
	at org.gradle.tooling.internal.consumer.DefaultBuildActionExecuter$1.run(DefaultBuildActionExecuter.java:57)
	at org.gradle.tooling.internal.consumer.connection.LazyConsumerActionExecutor.run(LazyConsumerActionExecutor.java:83)
	at org.gradle.tooling.internal.consumer.connection.ProgressLoggingConsumerActionExecutor.run(ProgressLoggingConsumerActionExecutor.java:58)
	at org.gradle.tooling.internal.consumer.async.DefaultAsyncConsumerActionExecutor$1$1.run(DefaultAsyncConsumerActionExecutor.java:55)
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	... 3 more
Caused by: java.lang.UnsupportedClassVersionError: com/github/rholder/gradle/acumen/api/AcumenTreeModel : Unsupported major.minor version 51.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:621)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:471)
	at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:259)
	at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:255)
	at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:231)
	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:124)
	... 31 more
```

**Comment by qmg-gkalogiros** [comment]
+1

**Comment by benmarch** [comment]
+1

**Comment by zzz40500** [comment]
+1

**Comment by chenhai1030** [comment]
+1

**Comment by rholder** [comment]
I have seen this happen when the IntelliJ runtime is running under a JDK 1.6. Older versions of IntelliJ shipped with support only for JDK 1.6 for their runtime on OSX because of a lack of high resolution screen support in JDK 1.7. Installing later versions of the JDK does not update the runtime Intellij chooses to use (and the one that `gradle-view` also uses). Try updating to the latest version of IntelliJ Ultimate/Community and make sure you're using an up to date JDK.

Please open another ticket if this problem persists.
